xcm/pallet-xcm/src/lib.rs

```diff
@@ -126,89 +126,67 @@ pub mod pallet {
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
 		/// Execution of an XCM message was attempted.
-		///
-		/// \[ outcome \]
-		Attempted(xcm::latest::Outcome),
+		Attempted { outcome: xcm::latest::Outcome },
 		/// A XCM message was sent.
-		///
-		/// \[ origin, destination, message \]
-		Sent(MultiLocation, MultiLocation, Xcm<()>),
+		Sent { origin: MultiLocation, destination: MultiLocation, message: Xcm<()> },
 		/// Query response received which does not match a registered query. This may be because a
 		/// matching query was never registered, it may be because it is a duplicate response, or
 		/// because the query timed out.
-		///
-		/// \[ origin location, id \]
-		UnexpectedResponse(MultiLocation, QueryId),
+		UnexpectedResponse { origin: MultiLocation, id: QueryId },
 		/// Query response has been received and is ready for taking with `take_response`. There is
 		/// no registered notification call.
-		///
-		/// \[ id, response \]
-		ResponseReady(QueryId, Response),
+		ResponseReady { id: QueryId, response: Response },
```

Comment:
There's loads of `id`s in this world. I think it's really helpful to prefix it with the type of id it is. In this case `query_id`. (Shawn has done this in the prior PR for an `index` field and I think that's a good shout. There's lot's of indices too!)